src/query/ast/src/parser/expr.rs

```diff
@@ -1003,11 +1008,18 @@ pub fn expr_element(i: Input) -> IResult<WithSpan<ExprElement>> {
         rule! {
             #function_name
             ~ "(" ~ DISTINCT? ~ #comma_separated_list0(subexpr(0))? ~ ")"
+            ~ (( IGNORE | RESPECT ) ~ NULLS)?
             ~ (OVER ~ #window_spec_ident)
         },
-        |(name, _, opt_distinct, opt_args, _, window)| ExprElement::FunctionCall {
+        |(name, _, opt_distinct, opt_args, _, opt_ignore_null, window)| ExprElement::FunctionCall {
             func: FunctionCall {
                 distinct: opt_distinct.is_some(),
+                window_ignore_null: match name.to_string().as_str().to_lowercase().as_str() {
+                    "first_value" | "first" | "last_value" | "last" | "nth_value" => {
```

Comment:
Now we support ignore nulls option. But sum() ignore nulls will also parse success. How to only support these `"first_value" | "first" | "last_value" | "last" | "nth_value"` support respect/ignore nulls ? cc @andylokandy